### ISSUE TITLE
Problem with picture in picture on iOS (PIP)

### ISSUE THREAD

**Issue by marcorinck** [nchutchind/cordova-plugin-streaming-media#169]
**What version of Streaming-Meda-Cordova-Plugin are you using?**
2.2.0 

**What version of Cordova are you using?**
8.0.0

**What devices are affected?**
all iPads

**Please describe the issue in detail, with relevant code samples**
When the user has started the video on an iPad its possible that he taps on the PiP button to play video in a small window hovering over my app (Picture in Picture, floating video). This small window has 3 buttons: Play/Stop, Close and Restore the video to fullscreen.

Unfortunately the restore button just closes the small video window and does NOT restore the full screen video. If the user has switched to another app in the mean time, my app is restored successfully but the video window also closes. 

I don't know what happens here exactly. I don't get any callback that something happened with the video and can't do anything about it. 

As I understand the iOS AVPlayerViewController has an [option to disable Picture-In-Picture](https://developer.apple.com/documentation/avkit/avplayerviewcontroller/1615821-allowspictureinpictureplayback) but this option isn't available in this plugin. 

I have seen in past issues, that there is a problem that AVPlayerViewController doesn't has a callback when the user closes the video, but can you maybe make this option for this plugin so that the user can't enter PiP at all?

**Comment by omeasraf** [comment]
Also experiencing the exact same bug, any way to fix it?

**Comment by shamilovtim** [comment]
I can probably disable picture in picture if that solves the problem

**Comment by marcorinck** [comment]
@shamilovtim if you can make the above linked option accessible inside the javascript plugin it would definitely help me. I don't want to break something for others.

**Comment by marcorinck** [comment]
Yes, from my point of view the restore button is not able to do what its supposed to do when using this plugin currently. At all.

You can choose to ignore the problem (or don't see it in the first place, like I did, only my tester found this little button ...) because the main functionality works ... playing the video. But when you don't want to have a half-working button in your UI, disabling PiP is a solution for everyone. 

To have a working PiP would be of course much nicer, but I guess you'd have to do quite a bit of rework for this, because the plugin would need to provide a video control inside the app (not only full screen). But I can't ask for this of course and I don't need it either, currently.

Thanks for your help, @shamilovtim

**Comment by pepsipotty** [comment]
Hi has anyone figured a workaround for this? PIP in the latest build still seems to be problematic.
